android/src/fdroid/play/listings/es-ES/title.txt

```diff
@@ -1 +1 @@
-OrganicMaps: GPS sin conexión
+Mapa Orgánico GPS sin conexión
```

Comment:
That user submits lots of seemingly automated, low-quality translations without checking, and appears to be chasing after me.

Reply:
Can you ping him on Weblate?

Reply:
good afternoon:

I made a correction I don't know what you think. The problem with that string is that Organic Maps was listed separately, when the name of the application is OrganicMaps.

And when I saw it that way, I translated it, but now I think it is OrganicMaps, instead of separate.

The problem sometimes I have it in https://hosted.weblate.org/translate/organicmaps/fdroid-metadata/es/?q=state%3A%3Ctranslated&offset=2#suggestions if you look, I did not make that suggestion but as Organic Maps appears then when translating ....

the bad thing is to put Offline Organic Maps Hike&Bike in 30 characters. Because Hike&Bike can refer to a website or to biking and hiking. And I looked at https://hosted.weblate.org/translate/organicmaps/fdroid-metadata/es/?q=state%3A%3Ctranslated&offset=2#translations (other languages). And in no language appears reference to Hike&Bike. And the word Offline the RAE (Royal Spanish Academy) advises to use in Spanish good afternoon:

I made a correction I don't know what you think. The problem with that string is that Organic Maps was listed separately, when the name of the application is OrganicMaps.

And when I saw it that way, I translated it, but now I think it is OrganicMaps, instead of separate.

The problem sometimes I have it in https://hosted.weblate.org/translate/organicmaps/fdroid-metadata/es/?q=state%3A%3Ctranslated&offset=2#suggestions if you look, I did not make that suggestion but as Organic Maps appears then when translating ....

the bad thing is to put Offline Organic Maps Hike&Bike in 30 characters. Because Hike&Bike can refer to a website or to biking and hiking. And I looked at https://hosted.weblate.org/translate/organicmaps/fdroid-metadata/es/?q=state%3A%3Ctranslated&offset=2#translations (other languages). And in no language appears reference to Hike&Bike. And the word Offline the RAE (Real Academía Española) advises to use in Spanish sin conexión. 

So I don't know what you think:  Mapas sín internet OrganicMaps 

A hug

Reply:
Removing the space is not a good idea, the Maps should be the clear separate word. If you don't have any better ideas, Organic Maps offline Mapas can be a better option.